subsys/dfu/dfu_target/src/dfu_target_uart_cddl/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+set(py_dfu_uart_gen_cbor
+        python3
+        ${ZEPHYR_BASE}/../modules/lib/zcbor/zcbor/zcbor.py
```

Comment:
This assumes a constant path that can never change, use the ZEPHYR MODULE define for the module path

Reply:
done